vertx-web-client/src/main/java/io/vertx/ext/web/client/impl/cache/CacheControlDirectives.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
+ *
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  and Apache License v2.0 which accompanies this distribution.
+ *
+ *  The Eclipse Public License is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  The Apache License v2.0 is available at
+ *  http://www.opensource.org/licenses/apache2.0.php
+ *
+ *  You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.ext.web.client.impl.cache;
+
+public interface CacheControlDirectives {
```

Comment:
I think we should use an enum here that wraps the string value

Reply:
Done.